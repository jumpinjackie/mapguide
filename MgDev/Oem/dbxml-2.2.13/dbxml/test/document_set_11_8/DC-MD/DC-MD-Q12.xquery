for $a in collection("11.8.1.dbxml")/order
where empty($a/order_lines/order_line[2])
return
    <OneItemLine>
        {$a/@id}
    </OneItemLine>
