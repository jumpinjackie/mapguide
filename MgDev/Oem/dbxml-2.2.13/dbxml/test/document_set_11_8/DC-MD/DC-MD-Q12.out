<OneItemLine id="9"/><OneItemLine id="15"/><OneItemLine id="17"/>